CHANGELOG.md

```diff
@@ -3,6 +3,7 @@ CHANGELOG
 
 unreleased
 ----------
+- Add 3DS support for non-network tokenized Google Pay cards
```

Comment:
Do we think it's fine for 3ds to just start running for some Google Pay cards? Or should it be opt in instead? I'm conflicted about it.

Reply:
:+1: